frontend/src/components/charts/doughnut.js

```diff
@@ -41,9 +47,10 @@ class Doughnut extends Component {
                     </div>
                     <div className="footer">
                         <div className="legend">
-                            <div><i className="fa fa-circle text-info" /> { this.props.data.labels[0] } </div>
-                            <div><i className="fa fa-circle text-warning" /> { this.props.data.labels[1] } </div>
-                            <div><i className="fa fa-circle text-danger" /> { this.props.data.labels[2] } </div>
+                            <div><i className="fa fa-circle text-info" /> { this.props.data.labels[0] } <Badge>{this.props.data.datasets[0].data[0]}</Badge></div>
```

Comment:
I think the extra badges are rather obtrusive and clutter the charts a bit too much (especially because they are not aligned). I'm even considering removing the legend altogether to clean things up. It seems to take up a lot of space while you can just hover/click the chart to find out what's what and choosing the right colors (green, orange, red) will mostly speak for themselves. What do you think?